wpeverest/user-registration
# Enhancement - Checkbox setting replaced with toggle.

### All Submissions:



### Changes proposed in this Pull Request:
This replaces the checkbox settings with a toggle.


### How to test the changes in this Pull Request:

1. Go to the Global setting and verify if all check checkboxes are replaced with a toggle or not.


### Types of changes:




### Other information:




### Changelog entry

>Enhancement - Checkbox setting replaced with toggle.
